### PR TITLE
issue-7-delete-date | Angel Leung

### PR DESCRIPTION
**Pull request by angelsleung**
issue-7-delete-date

### CODE REVIEW

File: server/public/styles.css [hunk 1]
```diff
@@ -330,6 +331,60 @@ body {
   background-color: lightgray;
 }
 
+.edit-page {
+  /* display: flex; */
+
+  /* justify-content: center; */
+
+  /* padding-top: 1rem; */
```
Comment (thebearingedge) [inline server/public/styles.css]: 😱

File: server/public/styles.css [hunk 2]
```diff
@@ -330,6 +331,60 @@ body {
...
+}
+
+.edit-row {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  /* padding-right: 4rem; */
```
Comment (thebearingedge) [inline server/public/styles.css]: 😿

File: server/public/styles.css [hunk 3]
```diff
@@ -330,6 +331,60 @@ body {
...
+  padding: 3rem 0 1rem;
+}
+
+.delete-route {
+  color: #ffcb91;
+  font-weight: 500;
+  font-size: 1.4rem;
+  cursor: pointer;
+
+  /* padding: 1.4rem 1.4rem 1.4rem 5rem; */
```
Comment (thebearingedge) [inline server/public/styles.css]: :sob:

File: client/pages/route-details.jsx [hunk 4]
```diff
@@ -181,16 +181,18 @@ export default class RouteDetails extends React.Component {
...
               <div className="option-button">
                 <i className="share-icon fas fa-share" />
                 <span className="button-text">Share</span>
               </div>
             </a>
-            <div className="option-button">
-              <i className="edit-icon fas fa-edit" />
-              <span className="button-text">Edit</span>
-            </div>
+            <a className="option-link" href="#edit-route">
```
Comment (thebearingedge) [inline client/pages/route-details.jsx]: I noticed that you have a few `setRouteId` method in App, but when you are doing hash routing, you should be able to derive the id of a specific item from the `route.params`.

```jsx
<a className="option-link" href={`#edit-route?${this.state.route.id}`}>
```

Take a close look at how `react-hash-routing` works, the user should be able to refresh the screen and not lose their place in the app. For example going to a specific route should switch the used to `#route-details?routeId=something`. Each component should fetch the data it needs based on the params in the hash. 

At present, you have a large amount of your application's state in `App`, but take a look at `react-hash-routing` and `react-context`. The app only knows 2 things:

- who the user is (if there are separate user accounts)
- what page to show

Everything else should be handled by each screen independently.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: server/public/styles.css (hunk 1 above)
Comment (angelsleung) [reply server/public/styles.css]: Oh nooo 🙀 I was in such a rush to submit the PR to you before the end of the day that I forgot to double-check my CSS for the comments. Sorry! 😅

**Comment by angelsleung** [comment]
issue-7-delete-datev2

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**